AppWidget/app/src/main/java/com/example/android/appwidget/ItemsCollectionAppWidget.kt

```diff
@@ -0,0 +1,184 @@
+/*
```

Comment:
Can this widget just be merged into the grocery list widget above? Since the grocery widget also has Checkboxes, it should be sufficient to demonstrate the new compound buttons APIs

Reply:
This Widget demonstrates other features other than Checkboxes such as Radio Button, Switch, and simplified usage of ListView. 
Given that, I think it's more readable to have a separate Widget than combining them all.